ui/imports/shared/status/StatusGifPopup.qml

```diff
@@ -335,36 +330,36 @@ Popup {
                 property string lastHoveredId
 
                 StatusGifColumn {
-                    // Not Refactored Yet
-//                    gifList.model: RootStore.chatsModelInst.gif.columnA
+                    gifList.model: RootStore.gifColumnA
```

Comment:
I actually dont know what Column A B C means :(

Reply:
if you open the gif popup, you will see 3 columns of gif, those are the A,B,C. The reason it is 3 models is that they can vary in height